tests/test_bundle.py

```diff
@@ -100,7 +100,7 @@ async def test_ingress_relation(ops_test: OpsTest):
     TODO (https://github.com/canonical/istio-operators/issues/259): Change this from using a
      specific charm that implements ingress's requirer interface to a generic charm
     """
-    await ops_test.model.deploy(KUBEFLOW_VOLUMES, channel="latest/edge")
```

Comment:
Is this pin needed?  I'm not against it exactly, but usually our `main` CI pins against `latest`.  If there's a reason why `latest` doesn't work, we should add a comment explaining why and when we can revert back to `latest`